guacamole-ext/src/main/resources/org/apache/guacamole/protocols/telnet.json

```diff
@@ -66,6 +66,11 @@
                     "name"    : "backspace",
                     "type"    : "ENUM",
                     "options" : [ "", "127", "8" ]
+                },
+                {
+                    "name"  : "terminal-type",
+                    "type"    : "ENUM",
+                    "options" : [ "xterm", "xterm-256color", "vt220", "vt100", "ansi", "linux" ]
```

Comment:
As above - not sure about whether xterm should be supported, yet, but others look good.